### DISCUSSION TITLE
AV1 Support for Nvidia 40, AMD 7000 and Intel Arc A series

### DISCUSSION THREAD

**Discussion by SolidStateDj** [alvr-org/ALVR#1494] [General]
## Proposal
With the new(ish) wave of GPUs that are upon us, each bringing AV1 hardware acceleration support (albeit in their own unique and non-standardized ways), I just thought it might be worth adding to the discussion since it didn't already appear to be. 

The proposal is to simply add hardware (or even software) accelerated AV1 support to ALVR.

## Background
For those unfamiliar with the AV1 codec, it is able to capture/stream/portray significantly higher image quality than its H264 or H265 counterpart at the same bitrate, meaning it can either achieve better quality at the same bandwidth, or the same quality at a lower bandwidth. 

## Reasoning
Given the inherent desire/necessity for immersion in VR, image quality being a considerable factor, I think it only makes sense to implement this feature. Obviously, this also implies that latency is not at an ungodly amount, but from what I understand, this is a non-issue.

## Obstacles
The largest hurdle that I currently see is the availability of the necessary hardware, which I myself do not currently possess, not to mention that in order to most effectively support it, all 3 (or more) implementations of the feature would be necessary for the best experience, although I admit not truly necessary.

**Comment by zmerp** [comment]
The main problem is that Android does not support AV1 decoding, at least not before Android 13. All currently supported headsets don't run Android 13 and most run on the XR2 SoC which has no support for AV1. Hardware acceleration support is mandatory when working with embedded.

**Reply by SolidStateDj** [reply]
Alright, that is completely fair, I completely forgot about the client end. One day....
